### PR TITLE
ilcdn.fi

### PR DESCRIPTION
**Pull request by theel0ja**

### CODE REVIEW

**Comment by injust** [comment]
This site contains active mixed content, both CSS and JS.

**Comment by theel0ja** [comment]
Done, what should I do with `static.ilcdn.fi`?

File: src/chrome/content/rules/ilcdn.fi.xml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <ruleset name="ilcdn.fi">
         <target host="static.ilcdn.fi" />
```
Comment (injust) [inline src/chrome/content/rules/ilcdn.fi.xml]: Move this to a comment above the header, such as:

```
<!--
	Active mixed content:
		- static.ilcdn.fi
-->
```

Comment (theel0ja) [reply src/chrome/content/rules/ilcdn.fi.xml]: Done. https://github.com/EFForg/https-everywhere/pull/15266/commits/700e0ffb6135b719741147cacb958e79a4201b28

File: src/chrome/content/rules/ilcdn.fi.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Active mixed content:
+		- static.ilcdn.fi
+-->
+
+<ruleset name="ilcdn.fi">
+        <target host="ad.ilcdn.fi" />
+        <target host="img.ilcdn.fi" />
```
Comment (J0WI) [inline src/chrome/content/rules/ilcdn.fi.xml]: `assets.ilcdn.fi`
`frontpage-s3.ilcdn.fi`

File: src/chrome/content/rules/ilcdn.fi.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Active mixed content:
+		- static.ilcdn.fi
```
Comment (J0WI) [inline src/chrome/content/rules/ilcdn.fi.xml]: Since only `http://static\.ilcdn\.fi/$` seems to be affected an exclusion would be the better choice here.

**Comment by J0WI** [comment]
Please add a comment here and in #15265 to show the relation.

**Comment by J0WI** [comment]
> Please add a comment here and in #15265 to show the relation.
An example of this can be found in the Amazon rulesets.

**Pull request closed without merging**